src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -441,6 +484,52 @@ public void largeDataValidationTest(final String inputVcfName,
         System.out.println("Total Elapsed Time: " + (endTime - overallStartTime) / 1e9 + "s");
     }
 
+    @Test(dataProvider = "provideForNonTrivialLargeDataTest")
```

Comment:
Add a comment explaining how large the test is

Reply:
Fixed.